htdocs/compta/index.php

```diff
@@ -897,7 +906,7 @@
 		$var=false;
 		$num = $db->num_rows($resql);
 
-		print '<div class="div-table-responsive-no-min">';
+		//print '<div class="div-table-responsive-no-min">';
```

Comment:
Why removing this ?
This c=allow to be able to scroll right and left with the finger when width of table is too large on a smartphone

Reply:
![2017-11-27 17_19_54-espace comptabilite_tresorerie - firefox developer edition](https://user-images.githubusercontent.com/3624836/33276939-56f5236e-d397-11e7-841c-20a087b14149.png)
Tooltip is partially masked